### ISSUE TITLE
Protocol double click action

### ISSUE THREAD

**Issue by DStrelak** [I2PC/scipion#2018]
**Is your feature request related to a problem? Please describe.**
When handling the protocol, sometimes I want to edit it, but mostly I'm interested in the result.

**Describe the solution you'd like**
Until protocol successfully finishes, double clicking it should open edit window.
Once it is done, double clicking should open the results (i.e. perform _Analyze result_ button click)

**Describe alternatives you've considered**
I know that this is purely design decision, so feel free to completely ignore this request.

**Comment by delarosatrevin** [comment]
I agree that in some cases seems more natural to open the "Analyze results' action with double click..but then introduce two effects for the same action. It is also very subjective preference from one user to another. So, I'm not sure about a a general solution here.
